src/components/networks/networkList.jsx

```diff
@@ -61,7 +63,7 @@ export class NetworkList extends React.Component {
                                     <Text component={TextVariants.h2}>{_("Networks")}</Text>
                                 </CardTitle>
                                 <CardActions>
-                                    <CreateNetworkAction />
+                                    { unlocked ? <CreateNetworkAction /> : <></> }
```

Comment:
In javascript/react, instead of ternary operator and filling the other option with empty `<></>`, you can use also `&& condition `, which I think is what we should use here
```suggestion
                                    { unlocked && <CreateNetworkAction /> }
```
You can image it as `if` condition, where if unclocked is `true`, <CreateNetworkAction /> gets rendered

More here: https://react.dev/learn/conditional-rendering#logical-and-operator-

Reply:
oh yes that is true. thanks for the suggestion